JAGPropertyConverter/JAGPropertyConverter.m

```diff
@@ -638,4 +661,13 @@ - (BOOL)_isKeyPathKey:(NSString *)key {
     return [key rangeOfString:@"."].location != NSNotFound;
 }
 
+// clear property value (set property to nil or 0 if it is primitive data type)
+- (void)_clearObjectValue:(id)object forProperty:(JAGProperty *)property {
```

Comment:
`- (void)_clearValueOfObject:(id)object forProperty:(JAGProperty *)property`

may be more descriptive of the parameter...